app/src/main/res/layout/fragment_home.xml

```diff
@@ -7,16 +7,149 @@
+        android:text="@string/coin_state"
+        android:textAlignment="textStart"
+        android:textColor="@color/black"
+        android:textSize="24sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/egg_home" />
+
+    <ImageView
+        android:id="@+id/egg_home"
+        android:layout_width="450dp"
+        android:layout_height="540dp"
+
+        android:layout_marginBottom="96dp"
+        android:src="@drawable/ic_egg"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/main_screen"
+        android:layout_width="220dp"
+        android:layout_height="250dp"
+        android:layout_marginTop="60dp"
+        android:background="@drawable/home_screen_curve"
+        app:layout_constraintBottom_toBottomOf="@+id/egg_home"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@+id/egg_home" />
+
+    <TextView
+        android:id="@+id/gray_ground_img"
+        android:layout_width="195dp"
+        android:layout_height="120dp"
+        android:layout_marginTop="52dp"
+        android:background="@drawable/home_screen"
+        app:layout_constraintEnd_toEndOf="@+id/main_screen"
+        app:layout_constraintStart_toStartOf="@+id/main_screen"
+        app:layout_constraintTop_toTopOf="@+id/main_screen" />
+
+    <TextView
+        android:id="@+id/happiness_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="12dp"
+        android:fontFamily="@font/bungee"
+        android:text="@string/happiness_title"
+        android:textColor="@color/black"
+        android:textSize="12sp"
+        app:layout_constraintStart_toStartOf="@+id/main_screen"
+        app:layout_constraintTop_toTopOf="@+id/main_screen" />
+
+    <TextView
+        android:id="@+id/happiness_progressBar_container"
+        android:layout_width="170dp"
+        android:layout_height="12dp"
+        android:layout_marginTop="-8dp"
+        android:background="@drawable/happiness_progressbar_container"
+        app:layout_constraintStart_toStartOf="@+id/happiness_title"
+        app:layout_constraintTop_toBottomOf="@+id/happiness_title" />
+
+    <TextView
+        android:id="@+id/happiness_progressBar"
+        android:layout_width="70dp"
+        android:layout_height="12dp"
+        android:background="@drawable/happiness_progressbar"
+        app:layout_constraintStart_toStartOf="@+id/happiness_progressBar_container"
+        app:layout_constraintTop_toTopOf="@+id/happiness_progressBar_container"
+        tools:layout_editor_absoluteY="181dp" />
+
+    <TextView
```

Comment:
Just to have a reminder: my doubt here was that the shop buttons should be coded as "button", but it turns out that the button class extends the text view, modifying the style. So, the functionalities are the same and it is correct to use a text view for a button